app/routes/calendar.py

```diff
@@ -0,0 +1,60 @@
+from app.models.models import db, Calendar
+from flask_restx import Resource, Namespace, fields
+import datetime
+
+api = Namespace('calendar', description='Calendar operations')
+
+# expected information
+model = api.model("Calendar",
+                {
+                    "start_date": fields.Date(required=True, description="Calendar start date"),
+                    "end_date": fields.Date(required=True, description="Calendar end date"),
+                    "location_id": fields.Integer(required=True, description="Calendar location"),
+                    "user_id": fields.Integer(required=True, description="Calendar user"),
+                }
+)
+
+@api.route("/")
+class Calendars(Resource):
+    '''Get all Scheduled Calendar Ranges'''
+    def get(self, location_id):
+        '''Get all Calendar Bookings'''
+        dates = Calendar.query.filter_by(location_id=location_id).all()
+
+        data = [day.to_dictionary() for day in dates]
+        
+        return {"dates": data}
+
+    @api.expect(model)
+    def post(self, location_id):
+        '''Create a new calendar booking with the provided date range'''
+        data = api.payload
+        dates = Calendar.query.filter_by(location_id=location_id).first()
```

Comment:
`dates = Calendar.query.filter_by(location_id=location_id).all()` in stead of .first()

Reply:
Implemented requested change